packages/server/graphql/isValid.ts

```diff
@@ -0,0 +1,5 @@
+// this is useful anytime you call `loadMany` on a dataloader
+const isValid = <T>(obj: T | Error | undefined | null): obj is T => {
```

Comment:
nit - we put that kind of predicates in different directories and it's hard to find them, maybe we should put everything In `utils/predicates` file? or if we want to have a separate file per predicate `utils/predicates`?